frontend/storage/models/Image.ts

```diff
@@ -12,3 +13,5 @@ export class Image {
 		this.fileUrl = props.fileUrl;
 	}
 }
+
+export const undefinedCover = new Image({ id: -1, name: 'undefined cover', fileUrl: UndefinedCover });
```

Comment:
My personal preference is to use PascalCase for exported constants but a quick google tells me it's not a common way of doing things. 

Anyhow `defaultImage` would be a better fit in this case, because the goal is to use this image everywhere an image is missing.